sdk/servicebus/service-bus/tsconfig.json

```diff
@@ -34,6 +34,11 @@
     "resolveJsonModule": true
   },
   "compileOnSave": true,
-  "exclude": ["node_modules", "typings/**", "./samples/**/*.ts"],
+  "exclude": [
+    "node_modules",
+    "typings/**",
+    "./samples/**/*.ts",
+    "./test/perf/azure-sb-package/*.ts"
```

Comment:
Why are we specifically excluding this file?

Reply:
"azure-sb-package" folder has a package.json to install "azure-sb".

The other folders in the perf folder don't have this problem since we are using the existing ones.
   - "rhea-promise sample" imports "rhea-promise" - pointing to the "sdk\servicebus\service-bus\node_modules\rhea-promise"
   - "service-bus sample" imports src code.

Build kept failing because of this azure-sb sample.
 https://dev.azure.com/azure-sdk/public/_build/results?buildId=24643
 ![image](https://user-images.githubusercontent.com/10452642/57052281-333cbe80-6c3b-11e9-9ac4-d2ac77a71fbc.png)


To avoid build failure, I've excluded the sample in the tsconfig.json.


@ramya-rao-a  Is there a better way to avoid build failure?